FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,58 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url, completion: { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				do {
+					let items = try FeedImageMapper.map(data, response)
+					completion(.success(items))
+				} catch {
+					completion(.failure(Error.invalidData))
+				}
+			case .failure(_):
+				completion(.failure(Error.connectivity))
+			}
+		})
+	}
+}
+
+private class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+	}
+
+	private struct Item: Hashable, Decodable {
```

Comment:
There's no need for the `Item` to conform to `Hashable`, please remove the conformance

```suggestion
	private struct Item: Decodable {
```